addons/website/static/src/xml/website.editor.xml

```diff
@@ -22,6 +22,20 @@
                     '<span class="highlighted-text">#</span>' to link to an anchor.
                 </small>
             </div>
+            <we-row>
+                <we-title>⌙ Media</we-title>
+                <t t-set="fileName" t-value="widget.$link.attr('data-file-name')"/>
```

Comment:
Not a fan of
- Reading a long chain from `widget` (`widget.a.b.c`)
- Reading the DOM from the qweb

You can read the DOM from the JS so that this template has access to the information directly